lib/paperclip.rb

```diff
@@ -57,6 +57,8 @@
 require 'paperclip/filename_cleaner'
 require 'paperclip/rails_environment'
 require 'mime/types'
+require 'mimemagic'
+require 'mimemagic/overlay'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.